apps/word-cloud/src/App.js

```diff
@@ -75,6 +90,7 @@ class App extends React.Component {
       if (word && word.length > 2 && wordsMap[word] && !stopWords[word]) {
         words.push({ text: word, value: wordsMap[word] });
       }
+      return word;
```

Comment:
Added this to prevent a warning – arrow function did not return anything before.